app/controllers/add-recipe.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  newName: '',
+  newPhoto: '',
+  newServings: '',
+  newTime: '',
+  newIngredients: '',
+  newDirections: '',
+  newNotes: '',
+  newFavorite: '',
+
+  actions: {
+    createRecipe() {
+      let recipe = this.store.createRecord('recipe', {
+        // id: 1000,
+        name: this.get('newName'),
+        photo: this.get('newPhoto'),
+        servings: this.get('newServings'),
+        time: this.get('newTime'),
+        ingredients: this.get('newIngredients'),
+        directions: this.get('newDirections'),
+        notes: this.get('newNotes'),
+        favorite: this.get('newFavorite'),
+      })
+      debugger
+      recipe.save()
```

Comment:
Interesting. We could assign an id in the console but could only make a hard-coded id work for real...unless we used Date.now() which wasn't really what we wanted. Thanks for the feedback and I'll look into the generateIdForRecord() method.